src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/test_hdinsight_commands.py

```diff
@@ -587,3 +598,8 @@ def _with_relay_outbound_and_private_link():
                '--subnet /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers' \
                '/Microsoft.Network/virtualNetworks/fakevnet/subnets/default ' \
                '--resource-provider-connection Outbound --enable-private-link'
+
+    @staticmethod
+    def _with_compute_isolation():
+        return '--version 3.6 -l southcentralus ' \
+               '--enable-compute-isolation --workernode-size Standard_E8S_V3 --headnode-size Standard_E8S_V3'
```

Comment:
Can you provide any test with --host-sku argument

Reply:
> Can you provide any test with --host-sku argument

Sure, currently we do not  need to provide this parameter, backend will provide a default value for this.
Anyway, I will update the related test case. Thanks.

Reply:
updated.